pyprocessing/environment.py

```diff
@@ -5,10 +5,8 @@ def size(w, h):
     global pp
     global width
     global height
-    pp.namespace['width'] = w
-    pp.namespace['height'] = h
```

Comment:
These are required to set up the tkinter window (see pyprocessing.renderer.tkinter.window)